src/whitespace_linter.clj

```diff
@@ -12,6 +12,8 @@
 
 (set! *warn-on-reflection* true)
 
+(def windows? (str/starts-with? (System/getProperty "os.name") "Windows"))
```

Comment:
Maybe instead of checking `os.name` it makes more sense to check whether `java.io.File/separator` is a backslash. Who knows, Microsoft might decide they want to change the name of the next version of Windows to winOS or something like that next year.

Reply:
Thanks for the thought @camsaul!

Lemme rethink this generically...

We have asked the user to use forward slashes in paths and patterns.
When comparing paths we are scanning, we want to normalize to the forward slash separator.

I think I see whatcha mean.
We can instead do this: if `java.io.File/separator` is not forward slash: 
- then replace any `java.io.File/separator` in scanned paths with forward slash
- otherwise no need to change scanned paths

No longer any need to check `os.name`.

Cool, I'll give this change a go soon.

Reply:
Ok @camsaul, have pushed the change.
Lemme know what you think.

Reply:
Also manually tested with rewrite-clj on:
- macOS
- Windows
- Linux